examples/sod-mpi.py

```diff
@@ -51,17 +52,36 @@
 from mirgecom.initializers import SodShock1D
 from mirgecom.eos import IdealSingleGas
 
+from logpyle import (LogManager, add_general_quantities,
+        add_simulation_quantities, add_run_info)
+
+from mirgecom.logging_quantities import PhysicalQuantity, KernelProfile
+
 
 logger = logging.getLogger(__name__)
 
 
 @mpi_entry_point
-def main(ctx_factory=cl.create_some_context):
+def main(ctx_factory=cl.create_some_context, use_profiling=False, use_logmgr=False):
     """Drive the example."""
+    if use_logmgr:
+        logmgr = LogManager("sod1d.dat", "wu")  # , comm=...
+        add_run_info(logmgr)
+        add_general_quantities(logmgr)
+        add_simulation_quantities(logmgr)
+    else:
+        logmgr = None
```

Comment:
Why?

Reply:
logmgr is need below, as an argument to advance_state.